### PR TITLE
xds: allow publishing grpc-xds to maven local

### PR DESCRIPTION
**Pull request by dapengzhang0**

### CODE REVIEW

File: xds/build.gradle [hunk 1]
```diff
@@ -95,4 +95,4 @@ publishing {
         }
     }
 }
-[publishMavenPublicationToMavenRepository, publishMavenPublicationToMavenLocal]*.onlyIf { false }
+[publishMavenPublicationToMavenRepository]*.onlyIf { false }
```
Comment (sanjaypujare) [inline xds/build.gradle]: Is this change enough? I looked at `../build.gradle` and it has on lines 435-437:
```
    [publishMavenPublicationToMavenRepository, publishMavenPublicationToMavenLocal]*.onlyIf {
        !name.contains("grpc-gae-interop-testing") && !name.contains("grpc-xds")
    }
```

That needs fixing too, right? There could be other changes needed as well in the parent gradle file.

**Comment by dapengzhang0** [comment]
Only need change this subproject directory, because only this subproject was blacklisted.

File: xds/build.gradle (hunk 1 above)
Comment (dapengzhang0) [reply xds/build.gradle]: I don't find these lines in `../build.gradle`. Are you looking at master branch?

Comment (sanjaypujare) [reply xds/build.gradle]: You are right - I was looking at an older branch and it looks like these lines have been removed. Sorry.

**Pull request merged**